packages/core/src/services/workspaces/create.ts

```diff
@@ -26,7 +26,7 @@ export async function createWorkspace(
     await createMembership({ confirmedAt: new Date(), user, workspace }, tx)
     await createApiKey({ workspace }, tx)
 
-    await importDefaultProject({ workspace, user }, tx)
+    await importDefaultProject({ workspace, user }, tx).then((r) => r.unwrap())
```

Comment:
Wait. We are returning a result here. This is going to throw if fail. Return the result